packages/app/src/components/KubernetesFilters.tsx

```diff
@@ -55,7 +62,7 @@ const FilterSelect: React.FC<FilterSelectProps> = ({
       disabled={isLoading}
       variant="filled"
       w={200}
-      limit={20}
+      limit={100} // Show up to 100 search results
```

Comment:
this is the # of items shown in the dropdown. The data is loaded into the component, but only 100 is shown at any time.